Dockerfile

```diff
@@ -6,7 +6,7 @@ USER root
 
 # when changing the version, don't forget also to change the sha1 checksum
 ARG DISTRO_LOC=https://archive.apache.org/dist/spark/spark-2.2.0/spark-2.2.0-bin-hadoop2.7.tgz
-
+ENV TINI_VERSION v0.16.1
 ENV PATH="$PATH:/opt/spark/bin" \
     SPARK_HOME="/opt/spark"
```

Comment:
nit: why creating [new layer](https://docs.docker.com/engine/reference/builder/#env)?

Reply:
true, could merge that with the `ENV` statement below

Reply:
I notice that Labels and ENVs don't add the image size
```c00f7c43b211        31 hours ago         /bin/sh -c #(nop)  LABEL io.k8s.description=P   0 B                 
bd0ad3ab5aa1        31 hours ago         /bin/sh -c #(nop)  ENV TINI_VERSION=v0.9.0      0 B                 
6e967f1c3101        31 hours ago         /bin/sh -c #(nop)  ENV SPARK_HOME=/opt/spark    0 B ```
but i may be wrong.
I will merge the ENVs thanks